src/utils/proposals.ts

```diff
@@ -1,5 +1,29 @@
 import { FrontEndProposal } from '../interfaces/FrontEndProposal';
 
+/**
+ * Iterates through a list of fields on the proposal,
+ * in preferential order (most ot least), looking for desired data.
```

Comment:
typo in the comment here, i imagine it's supposed to say "most or least"?

Reply:
Ah, good eye! I meant `most to least`, and will fix.